sys/parsoid.js

```diff
@@ -415,6 +415,9 @@ class ParsoidService {
             path
         );
 
+        // HACK: T359509
+        parsoidReq.body = '{}';
+
         return hyper.get(parsoidReq)
```

Comment:
This seems super strange - POST requests requrie a valid body, but GET request should require *no* body.